package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "memento-client",
-  "version": "1.0.0",
```

Comment:
would you mind reverting this? i would rather use `npm version major` to bump + tag